aspnetcore/blazor/blazor-server-ef-core.md

```diff
@@ -202,6 +232,19 @@ The factory is injected into components and used to create new instances. For ex
 > [!NOTE]
 > `Wrapper` is a [component reference](xref:blazor/components/index#capture-references-to-components) to the `GridWrapper` component. See the `Index` component (`Pages/Index.razor`) in the [sample app](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/blazor/common/samples/3.x/BlazorServerEFCoreSample/BlazorServerDbContextExample/Pages/Index.razor).
 
+New <xref:Microsoft.EntityFrameworkCore.DbContext> instances can be created with a factory that integrates with [ASP.NET Core's Identity model](xref:security/authentication/customize_identity_model):
```

Comment:
Same changes as other version. The real problem is not specific to identity, but rather when there are multiple `DbContext` instances with different connection strings.

Reply:
```suggestion
New <xref:Microsoft.EntityFrameworkCore.DbContext> instances can be created with a factory that allows you to configure the connection string per `DbContext` (such as when you use [ASP.NET Core's Identity model])(xref:security/authentication/customize_identity_model)):
```